Gruntfile.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = function(grunt) {
+
+   grunt.initConfig({
+
+      pkg: grunt.file.readJSON('package.json'),
+
+      eslint: {
+         target: [ 'Gruntfile.js', 'index.js', 'node.js', 'node-tests.js' ],
```

Comment:
Sorry, thinking about this more, we should change it to a glob. Otherwise someone will add a new file and definitely won't think to add the file to this list.